.github/workflows/deploy.yml

```diff
@@ -40,7 +41,15 @@ jobs:
           room_id: ${{ secrets.MATRIX_ROOM_ID }}
           access_token: ${{ secrets.MATRIX_ACCESS_TOKEN }}
           server: ${{ secrets.MATRIX_SERVER }}
-          message: "⚠️ Warning metadata deployed but should be updated one more time! ⚠️"
+          message: "⚠️ Warning metadata deployed but should be updated one more time! ⚠️ ${{ env.GITHUB_WORKFLOW_URL }}"
```

Comment:
env.GITHUB_WORKFLOW_URL it is something that we should build by themselves, for example:
```
env:
      GITHUB_WORKFLOW_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
      
```
https://github.com/nova-wallet/omni-enterprise/blob/dev/.github/workflows/integration_test_runner.yaml#L57